subiquity/controllers/installprogress.py

```diff
@@ -87,18 +85,15 @@ def curtin_error(self):
         else:
             self.default()
 
-    def run_command_logged(self, cmd, logfile_location):
-        with open(logfile_location, 'wb', buffering=0) as logfile:
+    def run_command_logged_to_journal(self, cmd):
+        with journal.stream("curtin_output") as logfile:
             log.debug("running %s", cmd)
             cp = subprocess.run(
                 cmd, stdout=logfile, stderr=subprocess.STDOUT, stdin=subprocess.DEVNULL)
```

Comment:
could be also done as `systemd-run --wait cmd` which will run requested command as a separate temporary unit with logging to journal. This just an fyi, rather than actual suggestion to do it this way.